src/io/p5.TableRow.js

```diff
@@ -127,6 +248,39 @@ p5.TableRow.prototype.get = function(column) {
  *  @param  {String|Integer} column columnName (string) or
  *                                   ID (number)
  *  @return {Number}  Float Floating point number
+ * @example
+ * <div class="norender"><code>
+ * // Given the CSV file "mammals.csv" in the project's "assets" folder:
+ * //
+ * // id,species,name
+ * // 0,Capra hircus,Goat
+ * // 1,Panthera pardus,Leopard
+ * // 2,Equus zebra,Zebra
+ *
+ * var table;
+ *
+ * function preload() {
+ *   //my table is comma separated value "csv"
+ *   //and has a header specifying the columns labels
+ *   table = loadTable('assets/mammals.csv', 'csv', 'header');
+ * }
+ *
+ * function setup() {
+ *   var rows = table.getRows();
+ *   var minId = Infinity;
+ *   var maxId = -Infinity;
+ *   for (var r = 0; r < rows.length; r++) {
+ *     var id = rows[r].getNum('id');
+ *     minId = min(minId, id);
+ *     maxId = min(maxId, id);
+ *   }
+ *
+ *   print([minId, maxId]);
```

Comment:
just to simplify, could we write this:
`print("minimum id = "+minId+" maximum id = "+maxId)`